app/viewmodels/checkAnswers/AssessmentsSummary.scala

```diff
@@ -33,42 +33,53 @@ object AssessmentsSummary {
     assessment: CategoryAssessment,
     indexOfThisAssessment: Int,
     isReassessmentAnswer: Boolean
-  )(implicit messages: Messages): Option[SummaryListRow] = {
+  )(implicit messages: Messages): Option[SummaryListRow] =
+    if (isReassessmentAnswer) {
```

Comment:
A lot of duplicate code between the two cases. not too worried about that to be fair. If is an easy/nice/sensible way to reduce that great, otherwise don't worry.

Reply:
Refactored